src/axolotl/prompters.py

```diff
@@ -18,7 +18,7 @@ class AlpacaPrompter:
     prompt_style = None
 
     def __init__(self, prompt_style="instruct"):
```

Comment:
Is it possible to 
```suggestion
    def __init__(self, prompt_style=PromptStyle.instruct.value):
```

Then, we would not need it in the next line?

Reply:
it get's called with None in data.py
```
            d_prompt_style = d_type_split[1] if len(d_type_split) > 1 else None
...
ds_strategy = AlpacaPromptTokenizingStrategy(
                    AlpacaPrompter(d_prompt_style),
                    tokenizer,
                    cfg.train_on_inputs,
                    cfg.sequence_len,
                )
```

Reply:
Gotcha. I was thinking: would it be better to use a different config `d.prompt_style` instead of packing it with `d.type`?